tf2_geometry_msgs/CMakeLists.txt

```diff
@@ -30,6 +30,17 @@ ament_auto_find_build_dependencies(REQUIRED ${required_dependencies})
 if(BUILD_TESTING)
   find_package(ament_cmake_gtest REQUIRED)
   find_package(rclcpp REQUIRED)
+
+  find_package(ament_cmake_cppcheck REQUIRED)
```

Comment:
why don't you add common ? instead of run only 4 tests?

Reply:
cppcheck needs to be called manually (with the `LANGUAGE "c++"` option), because of the header file suffix being `.h`. Otherwise it would only recognize them as plain C headers and complain about non-C code.

Reply:
You should be able to configure this before. 

 - [CMakeLists.txt example](https://github.com/ros2/rosidl_typesupport_fastrtps/blob/master/rosidl_typesupport_fastrtps_cpp/CMakeLists.txt#L73-L74)
 

Some ament links: 
 - [cppcheck](https://github.com/ament/ament_lint/blob/master/ament_cmake_cppcheck/cmake/ament_cmake_cppcheck_lint_hook.cmake#L38-L41)
 - [Uncrustify](https://github.com/ament/ament_lint/blob/master/ament_cmake_uncrustify/cmake/ament_cmake_uncrustify_lint_hook.cmake#L28-L31)